src/main/kotlin/io/github/flank/gradle/tasks/FlankRunTask.kt

```diff
@@ -26,10 +26,14 @@ constructor(
   @get:Input
   val dry: Property<Boolean> = objectFactory.property(Boolean::class.java).convention(false)
 
-  @get:InputFile
+  @get:InputFiles
```

Comment:
if it's not optional, can it be `InputFile`?

Reply:
The property is not optional because it always has a value, which in the worst case is the convention `ftl-credentials.json`.
The `@InputFiles` annotation is required because the file referenced by the property might not exist. In this case, if we had `@InputFile` annotation instead, the build will fail.